conda/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: ts-genericcamera
+  version: {{ data.get('version') }}
+
+source:
+  path: ../
+
+build:
+  script: python -m pip install --no-deps --ignore-installed .
+  script_env:
+    - PATH
+    - PYTHONPATH
+    - LD_LIBRARY_PATH
+    - OSPL_HOME
+    - PYTHON_BUILD_VERSION
+    - PYTHON_BUILD_LOCATION
+    - LSST_DDS_DOMAIN
+
```

Comment:
There is no test section on this recipe. I imagine it is because it won't work with the basic conda package builder.

Reply:
Correct. Many packages that are needed by the unit tests cannot be installed via conda or pip but require code compilation. Also, the conda package needs to be built for arm64 and not for amd64.